packages/sdk/src/transactions/types.ts

```diff
@@ -0,0 +1,166 @@
+import type { Inscription, Ordinal } from "../inscription/types";
+
+// Types are based on bitcoin RPC
+// Refer to https://bitcoincore.org/en/doc/25.0.0/rpc/rawtransactions/decoderawtransaction/
+
+/**
+ * Transaction output (Vout)
+ */
+export type Vout = {
+  /**
+   * Value in BTC
+   */
+  value: number;
+
+  /**
+   * Index
+   */
+  n: number;
+
+  /**
+   * Ordinals (Ordit RPC)
+   */
+  ordinals: Ordinal[];
+
+  /**
+   * Inscriptions (Ordit RPC)
+   */
+  inscriptions: Inscription[];
+
+  /**
+   * Is spent (Ordit RPC), can be Outpoint `{txid}:{vout}`, or `false` if not spent
+   */
+  spent: string | false;
+
+  /**
+   * Public key script
+   */
+  scriptPubKey: {
+    /**
+     * Diassembly of the public key script
+     */
+    asm: string;
+
+    /**
+     * Inferred descriptor for the output
+     */
+    desc: string;
+
+    /**
+     * The raw public key script bytes, hex-encoded
+     */
+    hex: string;
+
+    /**
+     * Number of required signatures
+     *
+     * @deprecated since bitcoin-core 22.0.0 RPC - see https://github.com/bitcoin/bitcoin/pull/22650
+     */
+    reqSigs?: number;
```

Comment:
I don't think this will ever be returned by ordit-api

Reply:
Removed